frontend/catalyst/pennylane_extensions.py

```diff
@@ -41,6 +41,13 @@
 from pennylane import QNode, QueuingManager
 from pennylane.operation import Operator
 from pennylane.ops import Controlled
+
+# TODO: Enable when ready
+# from pennylane.ops.op_math.controlled import (
+#     _get_pauli_x_based_ops,
+#     _handle_pauli_x_based_controlled_ops,
+#     _try_wrap_in_custom_ctrl_op,
+# )
```

Comment:
What is the condition for this to be ready?

Reply:
Oh, it was a leftover. Not needed now.